### PR TITLE
Test update methods

### PR DESCRIPTION
**Pull request by J03lMATH**
Full Method/Class overhaul

### CODE REVIEW

**Review by J03lMATH — COMMENTED** [review]
Fully Rewritten

**Comment by J03lMATH** [comment]
FREEEDOOOMMM

File: srcCode/test/TestComputeCoordinator.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		char delimiter = ',';
+		
+		ComputingResult result = ce.compute(new ComputingRequest(inputConfig,
+				outputConfig, delimiter));
+
+        assert(result.equals(ComputingResult.SUCCESS));
+        file.deleteOnExit();
+    } */
+	@Test
+	public void testRun() throws IOException {
```
Comment (sbrandon96) [inline srcCode/test/TestComputeCoordinator.java:59]: Great job! Looks like a nice test.

**Review by sbrandon96 — APPROVED** [review]

**Review by Youssup — APPROVED** [review]
Project done

**Pull request merged**